README.md

```diff
@@ -40,6 +40,26 @@ module "postgresql_rds" {
 }
 ```
 
+### Note about Enhanced Monitoring Support
```

Comment:
The capitalization here seems inconsistent (no capitalization of "about"; capitalization of "Support"). I'd consider making both lowercase or uppercase.

Reply:
In general, I think that this section is supplying too much information to a module user:

- An error message for a scenario that we believe cannot occur (given the `var.monitoring_interval > 0` fix)
- Criticism of Terraform documentation
- A link to a dynamic issue listing

I would consider rewording this section so that it focuses on:

- Describing the conditional we applied (the first sentence does this)
- A brief overview of why (the second does this)
- A link to the most relevant issue in the AWS provider repository

Reply:
You're right, and I think your solution strikes a good balance of supplying just enough information.